Cargo.toml

```diff
@@ -15,7 +15,7 @@ protobuf-codec = ["protobuf-build/grpcio-protobuf-codec", "raft-proto/protobuf-c
 prost-codec = ["prost", "prost-derive", "bytes", "lazy_static", "protobuf-build/grpcio-prost-codec", "raft-proto/prost-codec", "grpcio/prost-codec"]
 
 [dependencies]
-protobuf = { git = "https://github.com/nrc/rust-protobuf", branch = "v2.8" }
+protobuf = "=2.8.0"
```

Comment:
duplicated `=` ?

Reply:
No, the `=` tells Cargo to use exactly 2.8.0, and not to use 2.8.1, which doesn't match the replace